artemis-server/src/main/java/org/apache/activemq/artemis/core/config/BridgeConfiguration.java

```diff
@@ -75,6 +76,8 @@
    // The bridge shouldn't be sending blocking anyways
    private long callTimeout = ActiveMQClient.DEFAULT_CALL_TIMEOUT;
 
+   private ComponentConfigurationRoutingType routingType = ComponentConfigurationRoutingType.valueOf(ActiveMQDefaultConfiguration.getDefaultDivertRoutingType());
```

Comment:
Should this not be default bridge routing type? (looks like picking up the divert default)

Reply:
Nice catch. Fixed.